src/Datadog.Trace/TracingProcessManager.cs

```diff
@@ -114,141 +114,145 @@ private static Task StartProcessWithKeepAlive(ProcessMetadata metadata)
             var path = metadata.ProcessPath;
             Log.Debug("Starting keep alive for {Process}.", path);
 
-            return Task.Run(
-                async () =>
+            return Task.Factory.StartNew(state => InternalStartProcessWithKeepAlive(state), metadata).Unwrap();
```

Comment:
As for the previous Task.Run, for something that is going to be called only once I'd rather pay the price of the closure rather than the added complexity of the Unwrap 😃

Reply:
I don't have a strong opinion on this, you're right is only called once. In the other side `Unwrap` is pretty simple nowdays (https://source.dot.net/#System.Private.CoreLib/TaskExtensions.cs,e4f7cc9af0c22301), Is not automatic as Task.Run (https://source.dot.net/#System.Private.CoreLib/Task.cs,5291).

Reply:
reverted on 2027883